Jenkinsfile

```diff
@@ -551,7 +561,8 @@ set -ex
 log_file=''' + "${bldlabel}.txt" + '''
 exec 3>&1 1>>${log_file} 2>&1
 ccache -s
-source /opt/rh/devtoolset-6/enable
+##source /opt/rh/devtoolset-6/enable
```

Comment:
Do we want to support CentOS 6 *and* CentOS7 or just one?

Reply:
this is for the CI build only - and it's centos 7.